docs/about/digital.md

```diff
@@ -102,21 +102,21 @@ To facilitate reuse and analysis of the Project's edited and encoded text of Ms.
 
 For persistent access, each essay has been assigned a Digital Object Identifier (DOI) and will also be archived in Columbia University [Academic Commons](https://academiccommons.columbia.edu/) and downloadable as standalone publications.
 
 Static Site Generation
 ------
 
-Most websites are typically created using a content management system (CMS) in which the site's content is kept in a database and rendered into HTML when requested. This approach often requires the employment of proprietary software as well as a large number of resources, institutional support, or constant maintenance. Static site generation, unlike CMS or database-driven projects, allows websites to be created with significantly fewer dependencies, reduces maintenance overhead, and minimizes the risk of technological obsolescence. Content is already in its final form (as static pages are created beforehand) and can be sent as-is to the user.
+Most websites are typically created using a content management system (CMS) in which the site's content is kept in a database and rendered into HTML when requested. This approach often requires the employment of proprietary software as well as a large number of resources, institutional support, or constant maintenance. Static site generation, unlike CMS or database-driven projects, allows websites to be created with significantly fewer dependencies, reducing maintenance overhead, and minimizing the risk of technological obsolescence. Content is already in its final form (as static pages are created beforehand) and can be sent as-is to the user.
 
-Static site generation thus offers a number of advantages. It allows individualized software decisions and better control and maintenance of the site. Furthermore, open-source web servers like Apache and Nginx can very efficiently serve large amounts of static data and respond with increased user traffic, leading to lower server costs. Because the technology is so fundemental to the web, techniques for serving static sites are likely to persist into the future as well as be universally understood by IT professionals tasked with keeping the assets online.
+Static site generation thus offers a number of advantages. It allows individualized software decisions and better control and maintenance of the site. Furthermore, open-source web servers like Apache and Nginx can very efficiently serve large amounts of static data and respond to increased user traffic, leading to lower server costs. Because the technology is so fundamental to the web, techniques for serving static sites are likely to persist into the future as well as be universally understood by IT professionals tasked with keeping the assets online.
```

Comment:
I believe there is a reason this is "respond with" - while grammatically in English this is usually "respond to," here it signifies that as traffic changes, the system is responsive

Reply:
DECISION: "respond to"

ALSO:
> Because this is a well-established, fundamental web technology, techniques for serving static sites are likely to persist into the future as well as be universally understood by IT professionals tasked with keeping the assets online.